numbast/src/numbast/tools/static_binding_generator.py

```
@@ -253,6 +223,7 @@ def from_params(
         separate_registry: bool = False,
         function_argument_intents: dict | None = None,
         mlir_backend: bool = False,
+        cuda_toolkit_include_paths: list[str] | None = None,
```

Ruff (0.16.3)

[error] Missing `from __future__ import annotations`, but uses PEP 585 collection

Add `from __future__ import annotations`

(FA102)